PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/AzureCoreToolsMissingNupkgInstaller.kt

```diff
@@ -106,7 +118,8 @@ class AzureCoreToolsMissingNupkgInstaller : StartupActivity {
                                 val text = LoadTextUtil.loadText(file, 4096)
 
                                 // Check for known marker words
-                                if (knownMarkerWords.any { text.contains(it, true) }) {
+                                val knownMarker = markerToTriggerMap.filter { text.contains(it.key, true) }.firstOrNull()
```

Comment:
Agh... feeling that we need RIDER-57928 is growing :) I'll add this now.

Reply:
Was thinking it would be nice if we could replace this with an editor notification "This trigger needs the package XYZ. Install / Don't show again"

(would still need RIDER-57928)